website/docs/r/vapp_vm.html.markdown

```diff
@@ -288,14 +290,44 @@ Changes are ignored on update. This part isn't reread on refresh. To manage inte
 * `storage_profile` - (Optional) Storage profile which overrides the VM default one.
 
 
-<a id="customization"></a>
+<a id="customization-block"></a>
 ## Customization
 
+When you customize your guest OS you can set up a virtual machine with the operating system that you want.
+
+vCloud Director can customize the network settings of the guest operating system of a virtual machine created from a
+vApp template. When you customize your guest operating system, you can create and deploy multiple unique virtual
+machines based on the same vApp template without machine name or network conflicts.
+
+When you configure a vApp template with the prerequisites for guest customization and add a virtual machine to a vApp
+based on that template, vCloud Director creates a package with guest customization tools. When you deploy and power on
+the virtual machine for the first time, vCloud Director copies the package, runs the tools, and deletes the package from
+the virtual machine.
+
+~> **Note:** The settings below work so that all values are inherited from template and only the specified fields are
+overridden with exception being `force` field which works like a flag.
+
 * `force` (Optional) **Warning.** `true` value will cause the VM to reboot on every `apply` operation.
 This field works as a flag and triggers force customization when `true` during an update 
 (`terraform apply`) every time. It never complains about a change in statefile. Can be used when guest customization
 is needed after VM configuration (e.g. NIC change, customization options change, etc.) and then set back to `false`.
 **Note.** It will not have effect when `power_on` field is set to `false`. See [example workflow below](#example-forced-customization-workflow).
+* `enabled` (Optional; *v2.7+*) `true` will enable guest customization which may occur on first boot or if the `force` flag is used.
+This option should be selected for **Power on and Force re-customization to work**. For backwards compatibility it is
+enabled by default when deprecated field `initscript` is used.
+* `change_sid` (Optional; *v2.7+*) Allows to change SID (security identifier). Only applicable for Windows operating systems.
```

Comment:
> Only applicable for Windows operating systems.

This is a useful comment. Are there more options below which should have it too?

Reply:
The customization is pretty much "mystified". Many of the fields may work or not work properly based on the guest operating system. What I did find is that `change_sid` is mentioned exclusively in vCD documentation - https://docs.vmware.com/en/vCloud-Director/9.7/com.vmware.vcloud.user.doc/GUID-BB682E4D-DCD7-4936-A665-0B0FBD6F0EB5.html